packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -5923,14 +5913,25 @@ class MouseRegion extends SingleChildRenderObjectWidget {
   ///  * This widget, which used to contain a pointer, has disappeared.
   ///
   /// The last case is the only case when [onExit] does not match an earlier
-  /// [onEnter].
-  /// {@macro flutter.mouseTracker.onExit}
+  /// [onEnter]. This design is because the last case is very likely due to
+  /// the widget being unmounted, at which time calling `setState` will throw
+  /// exceptions. There are a few ways to mitigate this limit:
+  ///
+  ///  * If the state of hovering is completely managed within a widget that
+  ///    unconditionally creates this [MouseRegion], then this will not be a
+  ///    concern, since after the [MouseRegion] is unmounted the state is no
+  ///    longer used.
+  ///  * Otherwise, the outer widget is very likely accessible to the
```

Comment:
I'm not sure what you mean by "accessible to the condition" (and it is a bit awkward to read)